src/pudl/transform/eia860.py

```diff
@@ -575,3 +575,202 @@ def _make_phone_number(col1, col2, col3):
     )
 
     return u_df
+
+
+@asset
+def boilers(raw_emission_control_strategies_eia860, raw_boiler_info_eia860):
```

Comment:
Much of this data is ultimately going to produce a table named `boilers_eia860` (as well as `boilers_entity_eia` which I think already existed) but that won't happen until after the entity resolution / harvesting step. The dataframe that's produced by this function is a pre-normalization version of the boiler data.

Reply:
- I would rename this asset `clean_boilers_eia860` to be consistent with the other pre-normalized eia860 tables. 
- I don't think this needs an `io_manager_key` because we are currently using the default IO manager for these pre-normalized tables as we don't have db schemas for them yet. 
- Yes we should update the doc strings of these assets that still refer to `eia_860_dfs ` and `eia_860_transformed_dfs`. I think `pudl.transform.eia923` doc strings to be updated as well. Maybe we should do this in a separate PR?
- I don't think we need to change anything with `boil_eia860` because it's just reading the table from the db.